config/config.go

```diff
@@ -26,10 +26,9 @@ type Config struct {
 	// todo maybe merge port into host as it's for AN
 	RPCHost string
 	// todo support also just specifying latest height
-	// InitHeight provides initial heights for EVM block height
-	// useful only on a cold-start with an empty database, otherwise
-	// should be avoided or an error will be thrown.
-	InitHeight uint64
+	// InitCadenceHeight provides initial heights for Cadence block height
+	// useful only on a cold-start with an empty database
+	InitCadenceHeight uint64
```

Comment:
Minor, wouldn't `InitFlowHeight` be more accurate here?

Reply:
I know you mentioned this before, and it's a valid point, but I believe we now agreed the "FVM execution" will be called Cadence env, and EVM will be called EVM env, we try to not use Flow because Flow is now everything, Cadence + EVM, and it means the network itself. I think its defined like this in the FLIP.

Reply:
Make sense then :+1: